frontend/src/prompted_readings/analysis_view.js

```diff
@@ -28,14 +28,32 @@ class AnalysisView extends React.Component {
     render() {
         if (this.state.analysis !== null) {
             const {  // object destructuring:
-                total_view_time,
+                total_view_time, all_responses
             } = this.state.analysis;
 
             return (
-                <div>
+                <div style = {{padding: '1em'}}>
                     <h1>Analysis of Student Responses</h1>
                     <h3>Total view time</h3>
                     <p>{total_view_time} seconds</p>
+
+                    <h3>Most Common Responses</h3>
```

Comment:
This is great. Please now refactor your specific analysis out into its own React component.